bindings/nodejs/examples/client/16-custom-plugin.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright 2021-2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Client, initLogger } from '@iota/sdk';
+require('dotenv').config({ path: '.env' });
+
+// Run with command:
+// yarn run-example ./client/04-get-output.ts
+
+// In this example we will get output from a known outputId
+async function run() {
+    initLogger();
+    if (!process.env.NODE_URL) {
+        throw new Error('.env NODE_URL is undefined, see .env.example');
+    }
+
+    const client = new Client({
+        // Insert your node URL in the .env.
+        nodes: [process.env.NODE_URL],
+        localPow: true,
+    });
+
+    try {
+        const nftId =
+            '0x0000000000000000000000000000000000000000000000000000000000000000';
```

Comment:
I think this never returns anything since the empty NFT ids aren't tracked, the indexer only tracks the actual NFT id
https://api.shimmer.network/api/indexer/v1/outputs/nft/0x0000000000000000000000000000000000000000000000000000000000000000